blackbox/docs/ingestion.txt

```diff
@@ -0,0 +1,52 @@
+.. _ingestion:
+
+===================
+Ingestion Framework
+===================
+
+Aside from using standard SQL to ingest data, CrateDB offers the possibility to
+ingest data through custom event-based ingestion endpoints. The ingestion
+framework of CrateDB provides highly available and distributed endpoints for
+custom protocols where thousands of events are being sent simultaneously,
+such as MQTT_. With CrateDB's ingestion framework no intermediate queues or
+brokers need to be set up.
+
+
+Terms And Concepts
+==================
+
+Ingestion is configured via *ingestion rules* which define how
+incoming data gets routed and stored. Such ingestion rules are bound
+to a specific *ingestion source*. Ingestion sources are provided by
+custom implementations, available as builtin components or as
+:ref:`CrateDB-Plugins <plugins>`.
+
+:Ingestion Rule:
+
+  An ingestion rule defines where to put the incoming data of a
+  specific ingestion source, based on predefined conditions. For more,
+  details see :ref:`administration-ingestion-rules`
```

Comment:
Here we could elaborate a bit on that the schema / payload mapping from source to target table is specific to the ingestion source implementation

Reply:
i renamed ingestion endpoint to ingestion source to make it coherent with the rest of the docs